src/layout.rs

```diff
@@ -590,6 +617,84 @@ fn try_split(area: Rect, layout: &Layout) -> Result<Rc<[Rect]>, AddConstraintErr
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::prelude::Constraint::*;
+
+    fn equal_splits(
+        equal_chunks: bool,
+        constraints: Vec<Constraint>,
+        target: Rect,
+    ) -> Vec<(u16, u16)> {
+        let layout = Layout::default()
+            .direction(Direction::Horizontal)
+            .constraints(constraints)
+            .chunk_size_preference(if equal_chunks {
+                ChunkSizePreference::Equal
+            } else {
+                ChunkSizePreference::Unconstrained
+            });
+        let chunks = layout.split(target);
+        chunks.iter().map(|r| (r.x, r.x + r.width)).collect()
+    }
+
+    #[test]
+    fn test_split_equally_in_underspecified_case() {
+        let target = Rect::new(100, 200, 10, 10);
+        assert_eq!(
+            equal_splits(false, vec![Min(2), Min(2), Min(0)], target),
```

Comment:
nit: Can this either pass the ChunkSizePreference directly (add `use ChunkSizePreference::*` to make it concise) or create two methods. Generally bool params mean that people have to scroll up to work out the meaning.

Reply:
Fixed in https://github.com/ratatui-org/ratatui/pull/395/commits/819ed54821b6c4abe2c5e8d826e1da34cf5e0d52.